### DISCUSSION TITLE
HELP, macOS arm Metal only render 120fps

### DISCUSSION THREAD

**Discussion by luojunyuan** [ppy/osu#25502] [Q&A] [answered]
Set frame limit do not take any effect.
<img width="494" alt="image" src="https://github.com/ppy/osu/assets/28323948/7328db69-d12f-41b4-ace9-7459dc719248">

<img width="331" alt="image" src="https://github.com/ppy/osu/assets/28323948/2894ae25-43e6-4af7-b1ea-f0ce68f3fd23">

**Comment by peppy** [comment, marked as answer]
This is intended as it gives best performance and lowest latency under metal. We'll change the visual display to better reflect this in the future.
